app/portainer/views/home/homeController.js

```diff
@@ -101,7 +101,7 @@ angular
       });
 
       try {
-        const tags = TagService.tags();
+        const tags = await TagService.tags();
```

Comment:
we could take it a step further and remove the variable
```suggestion
        $scope.tags = await TagService.tags();
```